types/braintree/index.d.ts

```diff
@@ -1685,4 +1685,31 @@ declare namespace braintree {
     export interface TooManyRequestsError extends Error {}
     export interface UnexpectedError extends Error {}
     export interface UpgradeRequired extends Error {}
+
+    /**
+     * Validation errors
+     */
+
+    export interface ValidationError {
+        attribute: string;
+        code: string;
+        message: string;
+    }
+
+    export interface ErrorCollectionsObject {
+        [key: string]: ValidationErrorsCollection;
+    }
+
+    export interface ValidationErrorsObject {
+        [key: string]: ValidationError[];
+    }
+
+    export interface ValidationErrorsCollection {
```

Comment:
Could add `length` to the interface as well for this PR.
```
length: number
```

Reply:
I've tried to execute code so that I got those validation errors and they had no length even if documentation said so. Looking at the source code I don't see how as this is object and not an array. https://github.com/braintree/braintree_node/blob/master/lib/braintree/validation_errors_collection.js

Reply:
Thanks for doing some sleuthing and following up. I agree.